specification/search/resource-manager/Microsoft.Search/stable/2020-03-13/search.json

```diff
@@ -1641,15 +1602,63 @@
           "externalDocs": {
             "url": "https://azure.microsoft.com/documentation/articles/search-sku-tier/"
           }
+        },
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "The identity of the resource."
         }
       },
       "allOf": [
         {
-          "$ref": "#/definitions/Resource"
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
         }
       ],
       "description": "Describes an Azure Cognitive Search service and its current state."
     },
+    "SearchService_Update": {
```

Comment:
I would suggest using a usual camel case for type name, e.g. `SearchServiceUpdate`. It is hard to tell how different generator for different language would handle the underscore.